### DISCUSSION TITLE
自定义适配器的参考文档

### DISCUSSION THREAD

**Discussion by enderTree** [nonebot/discussions#337] [问答] [answered]
自己想开发一个企业微信的适配器，想问下有无适配器开发的参考文档

**Comment by yanyongyu** [comment, marked as answer]
适配器的开发文档目前在计划中，暂无。你可以参考 [OneBot v11](https://github.com/nonebot/adapter-onebot/blob/master/nonebot/adapters/onebot/v11/adapter.py) 等现有适配器进行开发。以及新版编写中的文档 https://deploy-preview-1604--nonebot2.netlify.app/docs/advanced/adapter 。如果遇到问题欢迎在群里或者discussion提出。
